integration/init_app/integration_test.go

```diff
@@ -139,8 +113,14 @@ var _ = Describe("ship init replicated.app/...", func() {
 					err := cmd.Execute()
 					Expect(err).NotTo(HaveOccurred())
 
+					replacements := map[string]string{
+						"__upstream__":       strings.Replace(upstream, "&", "\\u0026", -1),
```

Comment:
this is a hack, and that's :+1:, but might be worth a comment about what this is doing. I get InstallationID and customerId, but the `\\u0026` might not be immediately obvious

Reply:
I'll add a comment for that, you're right about it not being obvious